modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/util/DDMFormInstanceFieldSettingsValidator.java

```diff
@@ -233,25 +232,19 @@ private Set<String> _getInvalidDDMFormFields(
 		Map<String, DDMFormField> ddmFormFieldsMap =
 			fieldDDMForm.getDDMFormFieldsMap(true);
 
-		Set<Map.Entry<DDMFormEvaluatorFieldContextKey, Map<String, Object>>>
-			entrySet = ddmFormFieldsPropertyChanges.entrySet();
+		for (Map.Entry<DDMFormEvaluatorFieldContextKey, Map<String, Object>>
+				entrySet : ddmFormFieldsPropertyChanges.entrySet()) {
 
-		Stream<Map.Entry<DDMFormEvaluatorFieldContextKey, Map<String, Object>>>
-			stream = entrySet.stream();
+			if (!MapUtil.getBoolean(entrySet.getValue(), "valid", true)) {
+				DDMFormEvaluatorFieldContextKey ddmFormFieldContextKey =
+					entrySet.getKey();
 
-		stream.forEach(
-			entry -> {
-				if (!MapUtil.getBoolean(entry.getValue(), "valid", true)) {
-					DDMFormEvaluatorFieldContextKey ddmFormFieldContextKey =
-						entry.getKey();
+				DDMFormField propertyFormField = ddmFormFieldsMap.get(
+					ddmFormFieldContextKey.getName());
 
-					DDMFormField propertyFormField = ddmFormFieldsMap.get(
-						ddmFormFieldContextKey.getName());
-
-					ddmFormFieldList.add(
-						_getFieldLabel(propertyFormField, locale));
-				}
-			});
+				ddmFormFieldList.add(_getFieldLabel(propertyFormField, locale));
```

Comment:
@LisztLiszt The variable is `ddmFormFieldList` but it's actually a HashSet. Please rename.